### ISSUE TITLE
SMTP not working

### ISSUE THREAD

**Issue by sdlopexvidri** [Ylianst/MeshCentral#1573]
Hi, I'm using meshcentral and after the las update, my administrator account show me the message "verify my account", but the smtp settings are not working, my local smtp server works fine but my verification email is not send to my account, tried even with a google account but my verification email keep not going out, so i need to tell te meshcentral server that my account is verified, with my other admin users cant see my first administrator user, so what can I do? I tried with mesctrl and nothing fix it, and with meshcentral resetaccount and even nothing happend.

Please help me.

here is my actual smtp configuration:
```
  "smtp": {
    "host": "mail.vidri.com.sv",
    "port": 25,
    "from": "noreply@vidri.com.sv",
    "tls": false
  }
```

**Comment by PathfinderNetworks** [comment]
I would think your SMTP server would require authentication to send emails on behalf of the noreply@vidri.com.sv account.  Your configuration shows that you aren't using any sender authentication at all.  You will need to verify if your SMTP server does, in fact, require sender authentication or not.
If it does require authentication then the config would look more like this (obviously with the username and password applicable to your setup).  Also, you are better off using TLS if your email server supports it. In that case the port would likely be 465 (or maybe 587) and TLS would be set to true. 

   "smtp": {
      "host": "mail.vidri.com.sv", 
      "port": 25, 
      "from": "noreply@vidri.com.sv", 
      "user": "noreply@vidri.com.sv", 
      "pass": "yourpassword", 
      "tls": false
}

**Comment by sdlopexvidri** [comment]
Hello, thanks PathfinderNetworks, here is the problem, my SMTP settings worked, my server doesn't requiere authentication, but with the update it didn't work anymore, so I tried with a Google account as the manual explains

"smtp": {
"host": "smtp.gmail.com",
"port": 587,
"from": "myuser@gmail.com",
"user": "myuser@gmail.com",
"pass": "mypassword",
"tls": false
}

But it still doesn't work. 

Thanks for your response

**Comment by PathfinderNetworks** [comment]
For the Gmail SMTP connection to work you do need to enable TLS.  So the config would look like this.  See if that does anything? If it does work with Gmail then you will know that the issue is something to do with how your own SMTP server accepts email.  

"smtp": {
"host": "smtp.gmail.com",
"port": 587,
"from": "myuser@gmail.com",
"user": "myuser@gmail.com",
"pass": "mypassword",
"tls": true
}

**Comment by Ylianst** [comment]
Also, if you can enable tracing for "Email/SMS traffic" in the "My Server / Trace" tab and try sending an email, send back any errors you see. Thanks.

![image](https://user-images.githubusercontent.com/1319013/86391476-b9dd9900-bc4e-11ea-9851-4858ddb547f0.png)

**Comment by sdlopexvidri** [comment]
![Captura desde gmail](https://user-images.githubusercontent.com/47534535/86393798-c8ca4880-bc5a-11ea-9dc5-7aac7fc49955.JPG)
Thanks Ylianst, here is my trace, I noticed username and password wa wrong, but happend that google has been blocked the access from my meshcentral server, just accept on apps not safe and ready, I got my verfication email.

Solved Thanks a lot Ylianst and PathfinderNetworks for your atention. You are great people.
